backend-beeyond/src/main/java/at/htl/beeyond/validation/NamespaceValidValidator.kt

```diff
@@ -0,0 +1,22 @@
+package at.htl.beeyond.validation
+
+import at.htl.beeyond.entity.User
+import java.nio.charset.Charset
+import javax.validation.ConstraintValidator
+import javax.validation.ConstraintValidatorContext
+
+class NamespaceValidValidator : ConstraintValidator<NamespaceValid, String> {
+    override fun isValid(namespace: String?, context: ConstraintValidatorContext): Boolean {
+        if (namespace == null) {
+            return false
```

Comment:
If the namespace has no value it should return `true`. The job of the NamespaceValid annotation is to see if the content is valid not if it exists or not.